tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/tests/parser/TclModelParserTest.xtend

```diff
@@ -141,7 +141,7 @@ class TclModelParserTest extends AbstractParserTest {
 			val emptyReferences = steps.assertSingleElement.contents.assertSize(3)
 			emptyReferences.forEach[
 				assertInstanceOf(StepContentElement)
-				value.assertNull
+				(it as StepContentElement).value.assertNull
```

Comment:
the previous step already performs the cast. you can write:
`assertInstanceOf(StepContentElement).value.assertNull` here or
```
   assertInstanceOf(StepContentElement) => [
      value.assertNull
   ]
```